### PR TITLE
Japan and Nederlands Bandplan Update

### PR DESCRIPTION
**Pull request by MadIceTea**
I will also take it upon myself to verify the accuracy of both - I'll be [contacting VERNON](veroncb@veron.nl) to verify the Nederland's one and have [this document](https://www.tele.soumu.go.jp/resource/e/search/myuse/use0303/batch.pdf) to verify Japan's information.

(However any changes necessary because of ^ will be separated to a new issue + update PR.)

### CODE REVIEW

**Comment by MadIceTea** [comment]
### Description

Added more information regarding Japan & Nederland's amateur radio service, including a novel addition of Japan's band-plan chart.

### Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] This change possibly requires a documentation update

### How has this been tested?

This has not been tested to work, and I can't verify if the flake8_py3 will work as expected since the CI/CD test is not visible to me.

### Checklist

- [x] Code reviewed by the author
- [] Code documented (comments or other documentation)
- [] Changes tested
- [-] `flake8` passes
- [-] `CHANGELOG.md` updated
- [x] Informative commit messages
- [] Issue exists for PR
- [x] Descriptive PR title

**Comment by classabbyamp** [comment]
Can you still make an issue for this? I'll look into why flake8 isn't running.

CHANGELOG.md should be updated. In the Unreleased section, add the appropriate new headers, then describe your changes.

**Comment by MadIceTea** [comment]
Yah, sure I'll go fix all this stuff and be back sometime between now and 3 days from now.

**Comment by MadIceTea** [comment]
Changelog made, I think it got auto-added to the PR.

**Comment by classabbyamp** [comment]
Here are SVGs of each page:
* https://i.qrm.cat/jp_bandplan_pg1.svg
* https://i.qrm.cat/jp_bandplan_pg2.svg

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+## [2.3.2] - 2020-04-18
+### Added
+- Japanese bandplan information to display with the `?plan jp` command
+### Fixed
+- Details to the Nederlands bandplan command to accurately represent VERNON (Nederlands ARRL equivalent organisation).
```
Comment (classabbyamp) [inline CHANGELOG.md:13]: Also, `Nederlands` -> `Netherlands` 2x in this line

**Comment by classabbyamp** [comment]
@MadIceTea have you had a chance to make the requested changes?

File: CHANGELOG.md (hunk 1 above)
Comment (MadIceTea) [reply CHANGELOG.md]: Fixed below.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+## [2.3.2] - 2020-04-18
+### Added
```
[the remark this answers is not in the document]
Comment (MadIceTea) [reply CHANGELOG.md:10]: Could this part be resolved by maintainers at the final? It should be easier for your team to decide the actual date / version number than my guess.

Thanks.

**Review by MadIceTea — COMMENTED** [review]
This appears to have been done correctly now, please let me know.

**Comment by MadIceTea** [comment]
Hi @classabbyamp 

> @MadIceTea have you had a chance to make the requested changes?

It's been a busy year, but it looks like I can keep a promise to myself to have this feature implemented, hopefully(?) by the end of [Hacktoberfest](https://hacktoberfest.digitalocean.com/).

Speaking of which, would one of the maintainers mind adding a `hacktoberfest-accepted` tag to this PR? Would be much appreciated.

I left most of @0x5c's change suggestion untouched as I figured that this was better resolved by the maintainers at the final, when merge time is ready.

Thanks.

File: CHANGELOG.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (classabbyamp) [reply CHANGELOG.md:10]: You should not be putting a specific version number and date. Put your changes under an `## [Unreleased]` header. See [here](https://github.com/miaowware/qrm2/pull/278/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edL7) for an example.

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,12 +6,15 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
```
Comment (classabbyamp) [inline CHANGELOG.md:8]:
```suggestion

```
unnecessary space

File: CHANGELOG.md [hunk 4]
```diff
@@ -129,7 +132,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ## 1.0.0 - 2019-07-31 [YANKED]
 
 
-[Unreleased]: https://github.com/miaowware/qrm2/compare/v2.4.1...HEAD
+[Unreleased]: https://github.com/miaowware/qrm2/compare/v2.4.2...HEAD
```
Comment (classabbyamp) [inline CHANGELOG.md:135]:
```suggestion
[Unreleased]: https://github.com/miaowware/qrm2/compare/v2.4.1...HEAD
```
should not have been changed

File: README.md [hunk 5]
```diff
@@ -32,6 +32,14 @@ $ run.sh
...
 Check out the [contribution guidelines](/CONTRIBUTING.md) for more information about how to contribute to this project.
 
+## Contributors
+
+[0x5c](https://github.com/0x5c)
+
+[bruceyang1998](https://github.com/bruceyang1998)
+
+[madicetea](https://github.com/madicetea)
+
```
Comment (classabbyamp) [inline README.md:42]:
```suggestion

```

This should not be included in this PR. An issue should be made and the change discussed before it gets potentially implemented.

File: resources/img/bandcharts/meta.json [hunk 6]
```diff
@@ -2,6 +2,7 @@
     "ca": ["ca.png", "Canada", "Amateur radio bands in Canada", "**This bandplan is incomplete**; some bands, like 630m, are simply not present. It also does not cover any band above 30MHz.", "[RAC 0-30MHz Band Plan](https://www.rac.ca/wp-content/uploads/files/pdf/RAC%20Bandplan%20December%201%202015.pdf)", "🇨🇦"],
     "cn": ["cn.png", "China", "Amateur radio bands in China", "", "Created by KN8U and NY7H", "🇨🇳"],
     "mx": ["mx.png", "Mexico", "Radio allocations in Mexico", "Full radio allocations chart for all services. No information specific to amateur radio is shown.", "Secretaría de Comunicaciones y Transportes (SCT) / Instituto Federal de Telecomunicaciones (IFT)", "🇲🇽"],
-    "nl": ["nl.png", "Netherlands", "Amateur radio bands in the Netherlands", "", "", "🇳🇱"],
+    "jp": ["jp.png", "Japan", "Amateur radio bands in Japan", "*If there is an interest, an update to this bandplan will have it translated to English*; and it is also pending checks against the latest update from the 総務省 (MIC) of Japan, March 2020", "[JARL Amateur-Use Bandplans](https://jarl.org/Japanese/A_Shiryo/A-3_Band_Plan/bandplan20150105.pdf)", "🇯🇵"],
```
Comment (classabbyamp) [inline resources/img/bandcharts/meta.json:5]:
```suggestion
    "jp": ["jp.png", "Japan", "Amateur radio bands in Japan", "Pending checks against the latest update from the 総務省 (MIC) of Japan, March 2020", "[JARL Amateur-Use Bandplans](https://jarl.org/Japanese/A_Shiryo/A-3_Band_Plan/bandplan20150105.pdf)", "🇯🇵"],
```
The comment about translation should probably not be shown to the user. We have other parts of the bot that are not in English (e.g. the Chinese bandplan).

**Review by classabbyamp — CHANGES_REQUESTED** [review]

**Comment by classabbyamp** [comment]
looking better, but still needs some changes

File: resources/img/bandcharts/meta.json (hunk 6 above)
Comment (MadIceTea) [reply resources/img/bandcharts/meta.json:5]: I suppose we could just open another PR for translations if one is done/found, so fair enough.

File: README.md (hunk 5 above)
Comment (MadIceTea) [reply README.md:42]: Got it. I'll separate it as its own issue first.

File: CHANGELOG.md (hunk 3 above)
Comment (MadIceTea) [reply CHANGELOG.md:8]: Hahaha whoops.

File: CHANGELOG.md (hunk 4 above)
Comment (MadIceTea) [reply CHANGELOG.md:135]: Sure. How is this determined anyways?

Comment (classabbyamp) [reply CHANGELOG.md:135]: it's done manually at release time currently

Comment (MadIceTea) [reply CHANGELOG.md:135]: Thanks.

**Comment by MadIceTea** [comment]
Changes made, incorporated one more wording change to the Changelog beyond what was suggested, based on a recent PR review from 0x5c in their project.

File: resources/img/bandcharts/meta.json (hunk 6 above)
Comment (0x5c) [reply resources/img/bandcharts/meta.json]: The call for translated versions *could* be part of the readme.

@classabbyamp Separate issue?

File: CHANGELOG.md [hunk 7]
```diff
@@ -5,13 +5,15 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
...
 ## [Unreleased]
-
+### Added
+- Japanese bandplan information displays when `?plan jp` command is run.
+### Fixed
+- Details to the Netherlands bandplan command to accurately represent VERNON (Netherlands ARRL equivalent organisation).
 
 ## [2.4.1] - 2020-10-06
 ### Changed
 - Bumped discord.py to 1.5.0
```
Comment (0x5c) [inline CHANGELOG.md:16]:
```suggestion


```
This line should not have been deleted

File: resources/img/bandcharts/meta.json [hunk 8]
```diff
@@ -2,6 +2,7 @@
...
     "cn": ["cn.png", "China", "Amateur radio bands in China", "", "Created by KN8U and NY7H", "🇨🇳"],
     "mx": ["mx.png", "Mexico", "Radio allocations in Mexico", "Full radio allocations chart for all services. No information specific to amateur radio is shown.", "Secretaría de Comunicaciones y Transportes (SCT) / Instituto Federal de Telecomunicaciones (IFT)", "🇲🇽"],
-    "nl": ["nl.png", "Netherlands", "Amateur radio bands in the Netherlands", "", "", "🇳🇱"],
+    "jp": ["jp.png", "Japan", "Amateur radio bands in Japan", "Pending checks against the latest update from the 総務省 (MIC) of Japan, March 2020", "[JARL Amateur-Use Bandplans](https://jarl.org/Japanese/A_Shiryo/A-3_Band_Plan/bandplan20150105.pdf)", "🇯🇵"],
+    "nl": ["nl.png", "Netherlands", "Amateur radio bands in the Netherlands", "*This version of this bandplan is no longer hosted on VERNON's website and it's accuracy cannot be guranteed.*", "[VERNON Global Band Plan van de Telecom Agency, Ministerie van Economische Zaken en Klimaat](https://www.veron.nl/downloads/brochures/)", "🇳🇱"],
```
Comment (0x5c) [inline resources/img/bandcharts/meta.json:6]:
```suggestion
    "nl": ["nl.png", "Netherlands", "Amateur radio bands in the Netherlands", "*This version of this bandplan is no longer hosted on VERNON's website and it's accuracy cannot be guaranteed.*", "[VERNON Global Band Plan van de Telecom Agency, Ministerie van Economische Zaken en Klimaat](https://www.veron.nl/downloads/brochures/)", "🇳🇱"],
```

**Review by 0x5c — CHANGES_REQUESTED** [review]
Just some minor mistakes, otherwise it's very nice 👌

Oh yeah, I still suggest compressing your commits.

File: resources/img/bandcharts/meta.json (hunk 6 above)
Comment (MadIceTea) [reply resources/img/bandcharts/meta.json]: I'm going to vote for separate issue.

**Comment by MadIceTea** [comment]
I'll need to spend some time with the compression of the commits tomorrow, I'll see what can be done in that direction.

File: CHANGELOG.md [hunk 9]
```diff
@@ -5,10 +5,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 
 ## [Unreleased]
+
```
Comment (0x5c) [inline CHANGELOG.md:8]:
```suggestion
```

File: CHANGELOG.md [hunk 10]
```diff
@@ -5,10 +5,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
...
+### Added
+- Japanese bandplan information displays when `?plan jp` command is run.
 ### Fixed
+- Details to the Netherlands bandplan command to accurately represent VERNON (Netherlands ARRL equivalent organisation).
 - eQSL, paper QSL, and Logbook of the World status in `?qrz` sometimes being shown incorrectly.
```
Comment (0x5c) [inline CHANGELOG.md:14]:
```suggestion


```

**Review by 0x5c — APPROVED** [review]

**Review by classabbyamp — APPROVED** [review]

**Comment by 0x5c** [comment]
We wanted to do the release today so we tried to compress the PR ourselves.
It failed since the git history had to many loops to `master`.
The authorship information has been preserved through the use of the `Co-authored-by:` tag in the commit message.
image

**Comment by MadIceTea** [comment]
Thank you. I didn't even know `loops to master` was a thing in Git until now, very informative as always!

**Comment by classabbyamp** [comment]
it was just because there were several merges going `master` to `MadIceTea:master`, which made it hard to just do a squash because the branch originally forked in April

**Comment by alexwiegmanpx** [comment]
I see, good to know. It has been a while since then, must remember to be more timely in the future.
